dataservice/util/data_gen/data_generator.py

```diff
@@ -330,9 +340,42 @@ def _create_genomic_files(self, total):
             gf_list.append(GenomicFile(**kwargs))
         return gf_list
 
+    def _create_workflows(self, total=2):
```

Comment:
can you make the total to take random number?

Reply:
👍